mail_private/README.rst

```diff
@@ -2,8 +2,6 @@
  Internal Messaging
 ====================
 
```

Comment:
Send private messages to specified recipients, regardless of who are in followers list.